nemo/collections/cv/modules/data_layers/mnist_datalayer.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+# =============================================================================
+# Copyright (c) 2020 NVIDIA. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+from os.path import expanduser
+
+from torchvision.datasets import MNIST
+from torchvision.transforms import Compose, Resize, ToTensor
+
+from nemo.backends.pytorch.nm import DataLayerNM
+from nemo.core.neural_types import AxisKind, AxisType, ClassificationTargetType, NeuralType, NormalizedImageType
+from nemo.utils.decorators import add_port_docs
+
+__all__ = ['MNISTDataLayer']
+
+
+class MNISTDataLayer(DataLayerNM, MNIST):
```

Comment:
another thought: can we have a "TorchvisionDataLayer" which will accept string as parameter (mnist, cifar10, etc?)